cmd/flux/reconcile.go

```diff
@@ -60,6 +60,7 @@ type reconcilable interface {
 	GetAnnotations() map[string]string
 	SetAnnotations(map[string]string)
 
+	hasReconciler() bool                 // does the object's current configuration have a reconciler to reconcile it?
```

Comment:
this is just a nit but a more apt name would be `isStatic`, since the static objects do have a reconciler to help with migration